smart-contracts/test/Lock/partialWithdraw.js

```diff
@@ -0,0 +1,81 @@
+
+const Units = require('ethereumjs-units')
+
+const deployLocks = require('../helpers/deployLocks')
+const Unlock = artifacts.require('../Unlock.sol')
+let unlock, locks
+
+const Promise = require('bluebird')
+if (typeof web3.eth.getBlockPromise !== 'function') {
+  Promise.promisifyAll(web3.eth, { suffix: 'Promise' })
```

Comment:
Thoughts?

Reply:
What is `promisifyAll`?

Reply:
I've removed this as I got it working without it. My understanding is that older versions of web3.eth didn't return promises, so this was a workaround to promisify all of them. It seems that's no longer the case with web3, so this is unneeded.